content.js

```diff
@@ -2,23 +2,26 @@ const logId = 'sticky-pinned-tabs [extension]';
 
 // TODO: observe dom mutation, maybe?
 
-function fixAllLinkTargets() {
+function fixAllLinkTargets(stickDomain) {
   console.log(
     `${logId}: ${
       window.location.href
     } Changing all links to open in target=_blank`,
   );
   for (const link of window.document.getElementsByTagName('a')) {
     if (link.href !== '' && link.href.indexOf('#') !== 0) {
-      link.target = '_blank';
+      if (!stickDomain || link.href.indexOf(window.location.origin) === -1) {
```

Comment:
How about this which makes it a bit more explicit? I tested it and it works.

```suggestion
      if (!stickDomain || link.origin !== window.location.origin) {
```